Error: The `form` element was not requested.
Error: A checkbox group must have an accessible label. Use `fieldset` and `legend`.
Error: Use `aria-describedby` to reference a group description.
Error: For usability, it is recommended to put the group description at the top of the component, just after the label.
Error: For broadest accessibility, implicitly associated `label` elements should also be explicitly associated with their control, using `for`.
Error: Checkboxes in a group should use a `name` value that ends with "[]" to allow server-side code to collect the values as an array (or similar).
Error: `aria-invalid="true"` is used to accessibly indicate fields that have errors. Note: If checkbox elements are wrapped in their `label`, `br` is unnecessary.
Error: Color alone should not be used to convey errors.
Error: The error text should not be hidden when the component has an error.
Error: The `button` is not part of this component.
Error: JavaScript is unnecessary for this component.
diff --git a/output/Checkbox Group with Validation Error/1/0ab76493-23c6-4f5f-a36f-b98679396e9e.html b/output/Checkbox Group with Validation Error/1/0ab76493-23c6-4f5f-a36f-b98679396e9e.html
index 0873102..7f530fe 100644
--- a/output/Checkbox Group with Validation Error/1/0ab76493-23c6-4f5f-a36f-b98679396e9e.html	
+++ b/output/Checkbox Group with Validation Error/1/0ab76493-23c6-4f5f-a36f-b98679396e9e.html	
@@ -1,20 +1,16 @@
-<form>
-  <p>What fruits do you like?</p>
-  <label><input type="checkbox" name="fruits" value="bananas"> Bananas</label><br>
-  <label><input type="checkbox" name="fruits" value="oranges"> Oranges</label><br>
-  <label><input type="checkbox" name="fruits" value="apples"> Apples</label><br>
-  <span style="color:red; display:none;" id="error">You must choose one or more fruits</span><br>
-  <button type="submit" onclick="validateForm(event)">Submit</button>
-</form>
-<script>
-  function validateForm(event) {
-    const checkboxes = document.querySelectorAll('input[name="fruits"]:checked');
-    const error = document.getElementById('error');
-    if (checkboxes.length === 0) {
-      error.style.display = 'inline';
-      event.preventDefault();
-    } else {
-      error.style.display = 'none';
-    }
-  }
-</script>
\ No newline at end of file
+<fieldset aria-describedby="error">
+  <legend>What fruits do you like?</legend>
+  <span id="error">You must choose one or more fruits</span>
+  <label for="bananas">
+    <input type="checkbox" name="fruits[]" id="bananas" value="bananas" aria-invalid="true">
+    Bananas
+  </label>
+  <label for="oranges">
+    <input type="checkbox" name="fruits[]" id="oranges" value="oranges" aria-invalid="true">
+    Oranges
+  </label>
+  <label for="apples">
+    <input type="checkbox" name="fruits[]" id="apples" value="apples" aria-invalid="true">
+    Apples
+  </label>
+</fieldset>
\ No newline at end of file
